platform/javascript/http_client_javascript.cpp

```diff
@@ -37,16 +37,31 @@ Error HTTPClient::connect_to_host(const String &p_host, int p_port, bool p_ssl,
 		WARN_PRINT("Disabling HTTPClient's host verification is not supported for the HTML5 platform, host will be verified");
 	}
 
+	port = p_port;
+	use_tls = p_ssl;
+
 	host = p_host;
-	if (host.begins_with("http://")) {
+
+	String host_lower = conn_host.to_lower();
```

Comment:
this broke javascript compilation :(, I  should have done a proper review when I had the time.

Reply:
Sorry! What exactly broke it? Should I PR a fix or will you do it? :)

Reply:
conn_host is not defined in JS, I made a PR now.
No problem :)